docs/UserGuide.md

```diff
@@ -474,13 +486,21 @@ Example:
 
 ### Accepted email formats
 
-NUScheduler checks the emails of your profiles against a list of valid NUS email domains.
+NUScheduler checks the following:
+
+* Local part (the characters in your email before `@`) of the emails must follow these rules: 
+
+  * Only alphanumeric characters and these special characters, `+`, `_`, `.`, `-` are allowed
```

Comment:
I might be wrong but they can't be consecutive as well right, or are you doing this in another pr

Reply:
You're right that the special characters cannot be consecutive in the current regex, I think we are not changing it since it is possibly an enhancement rather than a bug fix.

Reply:
Yup, good catch! I will add it in